### DISCUSSION TITLE
Text Selection via Highlighting

### DISCUSSION THREAD

**Discussion by rz1027** [qpdf/qpdf#669] [Q&A] [answered]
Isn't there a way to select text via highlighting rather than pressing shift and selecting a rectangular area with text in ??

Thanks

Using qpdf version 10.6.2 on Manjaro Linux Kernel 5.10

**Comment by jberkenbilt** [comment, marked as answer]
qpdf is a command-line tool and C++ library. Is it possible that you are having an issue with qpdfview?

**Reply by rz1027** [reply]
I am sorry you are right .I got mixed up.

**Reply by jberkenbilt** [reply]
No problem, happens all the time. :-)
